packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,11 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version XXX
+*Released*: XXX
+* add highlightLastSelectedRow to QueryGridPanel
+* change user-detail action to map to core.users instead of core.siteusers since site users require Project Admin or higher
```

Comment:
This was intentionally set as core.siteusers so that the details links in the SM > Administration > User Management users grid would go to a meaningful view in the SM app for each user. Since the core.users grid only has access to the users in the given project, using the core.siteusers grid for this scenario meant that more links would work. And since this page/grid was accessed by Application admins or Site admins, it was ok to use siteusers at that time.

Can you help me understand the scenario with this story that required this to be changed?

Reply:
No I actually don't need this change for my story. I was thinking maybe we want to show user link for non-admin, but seems that's not the case. reverted change